examples/v0/account_management/get_accessible_customers.py

```diff
@@ -0,0 +1,59 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""This example lists the resource names for the customers that the
+authenticating user has access to.
+
+The customer IDs retrieved from the resource names can be used to set
+the login-customer-id configuration. For more information see this
+documentation: https://developers.google.com/google-ads/api/docs/concepts/call-structure#login-customer-id
+"""
+
+from __future__ import absolute_import
+
+import argparse
```

Comment:
Do you still need to import `argparse` and `six`?

Reply:
Ack - nope - thank you for catching.